cmd/filebeat-keeper/main.go

```diff
@@ -78,11 +101,18 @@ func run(stopCh <-chan struct{}) error {
 		log.Infoln("Filebeat will not start until configmap being updated")
 	}
 
+	sec, err := strconv.ParseInt(livenessProbePeriodSeconds, 10, 64)
```

Comment:
consider putting it in a `init` function

Reply:
fixed